### ISSUE TITLE
update metamod v1314;

### ISSUE THREAD

**Issue by pubzoneweb** [roflmuffin/CounterStrikeSharp#590]
![Untitled](https://github.com/user-attachments/assets/d1603253-f985-4533-8d70-0a0562229a3e)

**Comment by 0x132406C** [comment]
yep

**Comment by pubzoneweb** [comment]
I've temporarily disabled Metamod just to keep the server running until a fix is available.

**Comment by raszyniaczek** [comment]
Got this same on Windows Dedicated 
![CSS Error](https://github.com/user-attachments/assets/b0092f72-cc24-4bbf-94e9-88df62e22600)

**Comment by stefanx111** [comment]
you must update your gamedata #584

**Comment by raszyniaczek** [comment]
How to downlad this file from this task?

**Comment by raszyniaczek** [comment]
Ok its coming now to start but dont started. I disable all metamod plugins and it doesnt work.

**Comment by NoSetViolin** [comment]
> Ok its coming now to start but dont started. I disable all metamod plugins and it doesnt work.

i think thats some problem in gamedata.json   
230   "IGameSystem_InitAllSystems_pFirst": {

**Comment by Avarets123** [comment]
I updated the metamod and also took the latest updated CSS, the error mentioned above appeared

**Comment by Avarets123** [comment]
@roflmuffin @KillStr3aK

**Comment by 0x132406C** [comment]
@roflmuffin

**Comment by CloudEMPG** [comment]
i have the same issue

**Comment by bili12685** [comment]
same issue on debian 11.1
[16:14:58.782] CSSharp: Failed to find signature for 'Host_Say'
[16:16:35.927] CSSharp: Failed to find address for IGameSystem_InitAllSystems_pFirst

**Comment by Avarets123** [comment]
solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work

**Comment by raszyniaczek** [comment]
> solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work

It doesnt work for me. I tried this on Windows Dedicated. 

When first round start my server got crash.

**Comment by qaziabdullah** [comment]
Tried the solution on Windows dedicated, it doesn't work. It will be better to wait for roflmuffin to update CSS.

**Comment by CloudEMPG** [comment]
> solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work

didnt work for me, still the same issue on linux server

**Comment by CloudEMPG** [comment]
> > solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work
> 
> didnt work for me, still the same issue on linux server

nevermind, i just needed to restart the server twice for some odd reason. but worked now, thanks

**Comment by Asser90** [comment]
> > > solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work
> > 
> > 
> > didnt work for me, still the same issue on linux server
> 
> nevermind, i just needed to restart the server twice for some odd reason. but worked now, thanks

Did you just change the gamedata.json? I'm having crashes on startup. 

Tried fresh install, with and without new gamedata.json.

Metamod alone works fine.

**Comment by CloudEMPG** [comment]
> > > > solution: follow the [link](https://github.com/roflmuffin/CounterStrikeSharp/pull/584) and make the changes thar are in the "Files changed" tab in your similar file and the server will work
> > > 
> > > 
> > > didnt work for me, still the same issue on linux server
> > 
> > 
> > nevermind, i just needed to restart the server twice for some odd reason. but worked now, thanks
> 
> Did you just change the gamedata.json? I'm having crashes on startup.
> 
> Tried fresh install, with and without new gamedata.json.
> 
> Metamod alone works fine.

yes, i justed edited the lines i needed to, in the gamedata.json
